core/templates/core/fonts/fonts.twig

```diff
@@ -24,38 +18,38 @@
 </div>
 
 <div class="su-serif">
-  <h2>Crimson Text</h2>
-  <p class="su-font-lead">Crimson Text ("Crimson") is a free and open-source text type family. The font is designed in the tradition of beautiful oldstyle type. It features six cuts characters for a wide range of European languages.</p>
-  <h1>{{ heading }} 1</h1>
-  <h2>{{ heading }} 2</h2>
-  <h3>{{ heading }} 3</h3>
-  <h4>{{ heading }} 4</h4>
-  <h5>{{ heading }} 5</h5>
-  <h6>{{ heading }} 6</h6>
-  {{ paragraph }}
+  <h2>Source Serif Pro</h2>
+  <p class="su-font-lead">Source Serif Pro is our primary serif font and should be used in headlines and body copy as a primary serif font in your web and social media communications. For situations where Source Serif Pro cannot be used, use Minion Pro or Times New Roman.</p>
   <h3>Style Variations</h3>
+  <div style="font-weight: 200; font-style: normal;">Extra-Light</div>
+  <div style="font-weight: 200; font-style: italic;">Extra-Light Italic</div>
+  <div style="font-weight: 300; font-style: normal;">Light</div>
+  <div style="font-weight: 300; font-style: italic;">Light Italic</div>
   <div style="font-weight: 400; font-style: normal;">Regular</div>
   <div style="font-weight: 400; font-style: italic;">Regular Italic</div>
   <div style="font-weight: 600; font-style: normal;">Semi-Bold</div>
   <div style="font-weight: 600; font-style: italic;">Semi-Bold Italic</div>
   <div style="font-weight: 700; font-style: normal;">Bold</div>
   <div style="font-weight: 700; font-style: italic;">Bold Italic</div>
+  <div style="font-weight: 900; font-style: normal;">Black</div>
+  <div style="font-weight: 900; font-style: italic;">Black Italic</div>
 </div>
 
 <div class="su-slab">
   <h2>Roboto Slab</h2>
-  <p class="su-font-lead">Roboto has a dual nature. It has a mechanical skeleton and the forms are largely geometric. At the same time, the font features friendly and open curves. While some grotesks distort their letterforms to force a rigid rhythm, Roboto doesn’t compromise,
-        allowing letters to be settled into their natural width. This makes for a more natural reading rhythm more commonly found in humanist and serif types.</p>
-  <h1>{{ heading }} 1</h1>
-  <h2>{{ heading }} 2</h2>
-  <h3>{{ heading }} 3</h3>
-  <h4>{{ heading }} 4</h4>
-  <h5>{{ heading }} 5</h5>
-  <h6>{{ heading }} 6</h6>
-  {{ paragraph }}
+  <p class="su-font-lead">Roboto Slab is a secondary font and should be used as a display font.</p>
   <h3>Style Variations</h3>
   <div style="font-weight: 100; font-style: normal;">Thin</div>
```

Comment:
It looks like there are only 3 font-weights for Roboto Slab in Figma (no "thin" variant) so perhaps Figma needs to be updated or do we need the "thin" variant for Decanter?

Reply:
I've verified with the designers that they DO NOT want the thin variation to be included. Good eye! I'll remove this variation file and font call.